traffic_ops/traffic_ops_golang/api/generic_crud.go

```diff
@@ -153,6 +162,48 @@ func GenericCreateNameBasedID(val GenericCreator) (error, error, int) {
 	return nil, nil, http.StatusOK
 }
 
+// GenericCreateNameBasedIDV5 does a Create (POST) for the given GenericCreatorV5 object and type. This exists as a generic function, for the use case of a single "name" key (not a numerical "id" key) and a lastUpdated field.
+func GenericCreateNameBasedIDV5(val GenericCreatorV5) (error, error, int) {
```

Comment:
GenericCreateNameBasedIDV5 is created to be used only by V5 apis and clients that makes use of tc.TimeRFC3339{} struct